pkg/tagger/collectors/kubernetes_main.go

```diff
@@ -47,20 +47,18 @@ func (c *KubeMetadataCollector) Detect(out chan<- []*TagInfo) (CollectionMode, e
 		return NoCollection, err
 	}
 	// if no DCA or can't communicate with the DCA run the local service mapper.
```

Comment:
is this comment still relevant?

Reply:
@CharlyF ok I see it's called in the `Pull` I was a bit confused, thought it was something to start like the DCA one